### PR TITLE
outcomes for Household survey

### PR DESCRIPTION
**Pull request by JacobHarrisonONS**

### CODE REVIEW

File: src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewSplitAddressProcessor.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
+package uk.gov.ons.census.fwmt.outcomeservice.converter.impl;
```
Comment (spdiaz1000) [inline src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewSplitAddressProcessor.java:1]: Go through the spec again
https://collaborate2.ons.gov.uk/confluence/display/CATD/FWMT+Interfaces+-+Outcomes+-+RM+-+New+Split+Address

Theres bits missing like a new Id is needed

File: src/main/resources/templates/payload/NEW_ADDRESS_REPORTED-event.ftl [hunk 2]
```diff
@@ -38,8 +38,10 @@
       "longitude" : "${address.longitude?string["0.#########"]}",
     </#if>
       "region" : "${region}",
+    <#if sourceCase != "NEW_SPLIT_ADDRESS">
```
Comment (spdiaz1000) [inline src/main/resources/templates/payload/NEW_ADDRESS_REPORTED-event.ftl:41]: check https://collaborate2.ons.gov.uk/confluence/display/CATD/FWMT+Interfaces+-+Outcomes+-+RM+-+New+Split+Address

File: src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewSplitAddressProcessor.java (hunk 1 above)
Comment (JacobHarrisonONS) [reply src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewSplitAddressProcessor.java:1]: I am confused? we generate a new ID when we pick up the message from preprocess and convert to superset

File: src/main/resources/templates/payload/NEW_ADDRESS_REPORTED-event.ftl (hunk 2 above)
Comment (JacobHarrisonONS) [reply src/main/resources/templates/payload/NEW_ADDRESS_REPORTED-event.ftl:41]: What are you referring too? region? cause region is removed in a bug fix

**Review by spdiaz1000 — CHANGES_REQUESTED** [review]

**Pull request merged**